images/build-timing-processor/app.py

```diff
@@ -0,0 +1,66 @@
+import json
+import re
+from pathlib import Path
+
+import yaml
+from fastapi import FastAPI, HTTPException, Request, Response
+from kubernetes import client, config
+
+config.load_incluster_config()
+
+batch = client.BatchV1Api()
+
+app = FastAPI()
+
+
+BUILD_STAGE_REGEX = r"^stage-\d+$"
+
+
+@app.post("/")
+async def gitlab_webhook_consumer(request: Request):
+    """
+    This endpoint receives the gitlab webhook for failed jobs and creates
```

Comment:
Why is it just failed jobs?  Or maybe the comment is just copy-pasted from the error classifier webhook?

Reply:
Yep my bad, that's a leftover comment.